src/utils.js

```diff
@@ -1,141 +1,111 @@
     }
+  }
 
-    // renderTodo function
-    static renderTodo = () => {
-      // select todo list
-      const todoListItems = document.querySelector('#todo-list');
-
-      // clean up tolistItems
-      todoListItems.innerHTML = '';
-
-      const list = this.getData();
+  // renderTodo function
+  static renderTodo = () => {
+    // select todo list
+    const todoListItems = document.querySelector('#todo-list');
 
-      // loop through the todo list
-      if (list) {
-        list.forEach((item) => {
-          // check if the task is completed
-          const checked = item.completed ? 'checked' : null;
+    // clean up tolistItems
+    todoListItems.innerHTML = '';
 
-          // create an li element with class="item"
-          const li = document.createElement('li');
-          li.setAttribute('class', 'item');
+    const list = LocalStorage.getData();
 
-          // if task completed then add checked class to the li to cross over the activity
-          if (checked === 'checked') {
-            li.classList.add('checked');
-          }
+    // loop through the todo list
+    if (list) {
+      list.forEach((item) => {
+        // check if the task is completed
+        const checked = item.completed ? 'checked' : null;
 
-          // add data, checkbox and delete button to li
-          li.innerHTML = `<input type = "checkbox" class="checkbox" ${checked}> ${item.description} <button class="edit"><i class="fa fa-edit" aria-hidden="true"></i></button> <button class="delete"><i class="fa fa-trash" aria-hidden="true"></i></button>`;
+        // create an li element with class="item"
+        const li = document.createElement('li');
+        li.setAttribute('class', 'item');
 
-          todoListItems.appendChild(li);
-        });
-      }
-
-      // select delete from the list
-      const deleteTodo = document.querySelectorAll('.delete');
-      deleteTodo.forEach((btn, index) => {
-        btn.addEventListener('click', () => {
-          this.remove(index);
-        });
-      });
+        // if task completed then add checked class to the li to cross over the activity
+        if (checked === 'checked') {
+          li.classList.add('checked');
+        }
 
-      // select edit from the list
-      const edit = document.querySelectorAll('.edit');
-      edit.forEach((btn, index) => {
-        btn.addEventListener('click', () => {
-          this.edit(index);
-        });
-      });
+        // add data, checkbox and delete button to li
+        li.innerHTML = `<input type = "checkbox" class="checkbox" ${checked}> ${item.description} <button class="edit"><i class="fa fa-edit" aria-hidden="true"></i></button> <button class="delete"><i class="fa fa-trash" aria-hidden="true"></i></button>`;
 
-      const checkmarked = document.querySelectorAll('.checkbox');
-      checkmarked.forEach((btn, index) => {
-        btn.addEventListener('click', () => this.completedCheck(index));
+        todoListItems.appendChild(li);
       });
```

Comment:
- [ ] It has an event listener on the checkbox (change). So kindly implement it

Reply:
This line of code is not there with this branch. The completedCheck() method is implemented in Status class and called in utils.js File.